pkg/monitor/cluster/arooperatorheartbeat.go

```diff
@@ -0,0 +1,28 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"strconv"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (mon *Monitor) emitAroOperatorHeartbeat(ctx context.Context) error {
+	aroDeployments, err := mon.cli.AppsV1().Deployments("openshift-azure-operator").List(metav1.ListOptions{})
+
+	if err != nil {
+		return err
+	}
+
+	for _, d := range aroDeployments.Items {
```

Comment:
at the moment this will not emit an unhealthy metric when an expected deployment is missing, and it should.
Once rebased on #1064, specifically check for the deployments you expect to exist and emit metrics accordingly.

Reply:
Done, added a map with both deployments, so the metric will be emitted for them every time.